src/components/ExecutorDetails.js

```diff
@@ -0,0 +1,311 @@
+                    <CardHeader className='ins-m-card__header-bold'>
+                        <Button
+                            variant='link' onClick={ () => downloadPlaybook(remediation.id) }>
+                            Download Playbook
+                        </Button>
+                    </CardHeader>
+
+                    <CardBody>
+                        { InventoryTable && <InventoryTable
+                            ref={ inventory }
+                            items={ orderBy(systems, [ s => getSystemName(s), s => s.id ]) }
+                            onRefresh={ onRefresh }
+                            page={ page }
+                            total={ systems.length }
+                            perPage={ pageSize }
+                            tableProps={ { onSelect: undefined } }
+                            expandable
+                            onExpandClick={ (_e, _i, isOpen, { id }) => {
+                                if (isOpen) {
+                                    getPlaybookRunSystemDetails(remediation.id, run_id, id);
+                                    setRefreshInterval(setInterval(() => getPlaybookRunSystemDetails(remediation.id, run_id, id), 10000));
+                                }
+                                else {
+                                    clearInterval(refreshInterval);
+                                }
+
+                                onCollapseInventory(isOpen, id);
+
+                            } }
+                        /> }
+                    </CardBody>
+                </Card>
+            </Stack>
+        </Main>,
+        success: <Main>
+            <Stack gutter="md">
+                <Card>
+                    <CardHeader className='ins-m-card__header-bold'>
+                        <Button
+                            variant='link' onClick={ () => downloadPlaybook(remediation.id) }>
+                            Download Playbook
+                        </Button>
+                    </CardHeader>
+
+                    <CardBody>
+                        { InventoryTable && <InventoryTable
+                            ref={ inventory }
+                            items={ orderBy(systems, [ s => getSystemName(s), s => s.id ]) }
+                            onRefresh={ onRefresh }
+                            page={ page }
+                            total={ systems.length }
+                            perPage={ pageSize }
+                            tableProps={ { onSelect: undefined } }
+                            expandable
+                            onExpandClick={ (_e, _i, isOpen, { id }) => {
+                                getPlaybookRunSystemDetails(remediation.id, run_id, id);
+                                onCollapseInventory(isOpen, id);
+                            } }
+                        /> }
+                    </CardBody>
+                </Card>
+            </Stack>
+        </Main>,
+        failure: <Main>
```

Comment:
even on failure let's show the inventory table so that users have a chance to see what went wrong